sdcm/coredump.py

```diff
@@ -202,12 +202,13 @@ def extract_info_from_core_pids(
     def _upload_coredump(self, core_info: CoreDumpInfo):
         coredump = core_info.corefile
         coredump = self._pack_coredump(coredump)
-        base_upload_url = 'upload.scylladb.com/%s/%s'
+        base_upload_url = 'https://upload.scylladb.com/%s/%s'
```

Comment:
I checked the history of scylla-docs and I don't see any change made there in this area.
Could it be that it's a change made by GCP?
Are you sure this is the root cause of the failure?

Reply:
We have some kind of proxy infront of GCP.
I'm guessing @amnonh knows more about it.

Could be there's a GCP change, but yes I'm sure that anonymous upload only works via https now. (you can cross check it)

Reply:
I missed the part we do it anonymously.
But I still don't understand how other tests were able to upload....

Reply:
26 days ago it was working (on SCT master branch): 
https://github.com/scylladb/scylla/issues/10553

and then in the last ~10 days, it's missing again and again:
https://github.com/scylladb/scylla/issues/10687
https://github.com/scylladb/scylla/issues/10735
https://github.com/scylladb/scylla/issues/10741